Analysis/interface/GenLepton.h

```diff
@@ -48,6 +48,7 @@ class GenParticle {
 
     int pdgId{0};
     int charge{0};
+    int status{0};
```

Comment:
`status == 8` for geant particles is configurable and can be different for other productions. Better to have `bool fromGeant;` instead of `status` at the level of `GenParticle` class.